lib/web_ui/lib/src/engine/scene_view.dart

```diff
@@ -305,23 +305,28 @@ final class PlatformViewContainer extends SliceContainer {
   @override
   void updateContents() {
     assert(_styling != null);
-    assert(_size != null);
+    assert(_bounds != null);
     if (_dirty) {
       final DomCSSStyleDeclaration style = container.style;
       final double devicePixelRatio = EngineFlutterDisplay.instance.devicePixelRatio;
-      final double logicalWidth = _size!.width / devicePixelRatio;
-      final double logicalHeight = _size!.height / devicePixelRatio;
+      final double logicalWidth = _bounds!.width / devicePixelRatio;
+      final double logicalHeight = _bounds!.height / devicePixelRatio;
       style.width = '${logicalWidth}px';
       style.height = '${logicalHeight}px';
       style.position = 'absolute';
 
-      final ui.Offset? offset = _styling!.position.offset;
-      final double logicalLeft = (offset?.dx ?? 0) / devicePixelRatio;
-      final double logicalTop = (offset?.dy ?? 0) / devicePixelRatio;
+      final PlatformViewPosition position = PlatformViewPosition.combine(
```

Comment:
Unable to comment on the internals of this class, so commenting here. Just a nit that we should probably put `transform` first and `offset` second in the order of fields, arguments, and various expressions that involve both because transform is applied first and offset is applied second. For example, if you have a transform layer and you do `addPlatformView(offset)`, you first apply the transform and then the offset. Unless I'm misunderstanding what these fields do.

Reply:
This is an "either-or" class, where either the offset or the transform are specified but never both. If there is an offset and a transform that are combined, they just become a transform only (the offset is applied properly to the transform). As such, there is no ordering implied here within this class.